### ISSUE TITLE
Need Usage Guide + Executable

### ISSUE THREAD

**Issue by PushpenderIndia** [oSumAtrIX/ThemidaStringDecryptor#1]
Hi mate! 
Can you please tell me how to use this tool, I am doing some test on a malware, it is packed with Themida Winlicense 3.0.8

I am able to unpack themida using Megadumper, and then fixed the assembly using Universal_Fixer

But the exe is obfuscated, so i want to try this tool, can this tool help me deobfuscate the unpacked exe? 

Thanks in Advance 🙃

**Comment by oSumAtrIX** [comment]
You need to compile it. Clone the repo, inside the solution folder. Make sure you have the .Net 5 sdk installed and execute "dotnet build -c Release" inside that folder from your command line. You will find the assemblies inside the bin folder

**Comment by PushpenderIndia** [comment]
I am able to compile, but the exe is not working, that is why I am asking for ready to use executable, can you please guide me that how to use it, I have tried to drag and drop the exe on this software, but it didn't work for me

**Comment by oSumAtrIX** [comment]
You will need to read the stack trace by launching the program from the command line and passing your file to decrypt as a parameter.

**Comment by PushpenderIndia** [comment]
Can you please provide me the executable, because I have tried that also, I have executed the exe like this : ThemidaStringDecryptor.exe myexe.exe

But unfortunately it doesn't work for me

**Comment by DevX-Cipher** [comment]
this is the error you get Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at ThemidaStringDecryptor.Program.Main(String[] args) in C:\Users\Modz2014\Downloads\themida-string-decryptor-master\ThemidaStringDecryptor\Program.cs:line 31

**Comment by oSumAtrIX** [comment]
You did something wrong then

**Comment by DevX-Cipher** [comment]
thats just from downloading the project then building then try to run it

**Comment by oSumAtrIX** [comment]
Yes. You can only run it with a .NET module dump of Themida as an argument.

**Comment by DevX-Cipher** [comment]
yes it is im doing a test
